packaging/docker/run.sh

```diff
@@ -34,18 +34,6 @@ create_group_and_assign_to_user() {
 	fi
 }
 
-DOCKER_USR="netdata"
```

Comment:
Do not remove this variable, it breaks netdata.
We use it elsewhere too

Reply:
Got it. Didn't realize this is used to tell NetData to run as the `netdata` user. I'm pretty sure this is actually why I'm seeing the issues with accessing the socket from within NetData, but not when I `exec`, since `exec` drops me into `root`.

Rather than chowning the Docker socket, I'm can add some instructions on how to pass in an existing `GID` that has access. The reason I believe this is safer is that access to the socket is essentially root access to the entire system. Any change of ownership is significant and should be done very deliberately by the user.

 Alternately, a variable could be used to tell NetData to run as `root` instead. There is no difference in safety since, as described earlier, access to the socket allows root escalation already.

Reply:
Hi, Please re-instate this variable. Netdata is designed to **not** run as root.
We control the level of privileged access is required on a per module basis.

Reference:
https://docs.netdata.cloud/docs/netdata-security/?origin_team=TCC5DL092#your-systems-are-safe-with-netdata